tests/class_test.py

```diff
@@ -0,0 +1,4 @@
+from privacy_evaluator.attacks.sample_attack import Sample_Attack
+
+test = Sample_Attack(0, 0, 0)
+test.perform_attack()
```

Comment:
It would be nice if you could use `pytest` for testing here so that we can run multiple tests in different files at the same time and get a better output. Have a look at this brunch: [marisanest:9-train-test-gap-metrics](https://github.com/marisanest/privacy-evaluator/tree/9-train-test-gap-metrics) there I already used it. You can simply run it with `pytest ./tests/*`

Reply:
I added a proper test and subfolder for it

Reply:
Can we maybe just squash and merge this? I would like to get the Inference Attack done before our meeting tomorrow :)

Reply:
unlikely that all aprove unitil then, but i think you can just merge the changes from this PR into your branch and go from there